.github/workflows/build-assets.yml

```diff
@@ -219,7 +222,7 @@ jobs:
           repo: LN
           token: ${{ secrets.LNB_TOKEN }}
           tag: ${{ needs.preamble.outputs.internal_tag }}
-          artifacts: "${{ needs.preamble.outputs.folder_build }}/hpccsystems-platform-internal*.deb,${{ needs.preamble.outputs.folder_build }}/hpccsystems-platform-internal*.rpm"
+          artifacts: "${{ needs.preamble.outputs.folder_build }}/hpccsystems-platform-internal*.deb,${{ needs.preamble.outputs.folder_build }}/hpccsystems-platform-internal*.rpm,${{ needs.preamble.outputs.folder_build }}/hpccsystems-clienttools-internal*.deb,${{ needs.preamble.outputs.folder_build }}/hpccsystems-clienttools-internal*.rpm"
```

Comment:
Could we just use:  "${{ needs.preamble.outputs.folder_build }}/hpccsystems-*-internal*.deb,${{ needs.preamble.outputs.folder_build }}/hpccsystems-*-internal*.rpm"?

Otherwise looks good

Reply:
We could likely change it to just match "${{ needs.preamble.outputs.folder_build }}/*.deb" and "${{ needs.preamble.outputs.folder_build }}/*.rpm".  I just ran a test that dumped the contents of {{ needs.preamble.outputs.folder_build }} and it is of course clean for that run of the matrix. I'm trying to think how anything other than an internal build could be present, but can't.

Reply:
I updated my comment above as the wildcards were being converted by the formatter,

Reply:
Made the change you recommended. Tested using the tag internal_8.12.x-clienttools-3.